scripts/blueprints/stack-device.yaml

```diff
@@ -0,0 +1,26 @@
+selector:
+  sw.os: 1
+  hw.device-type: 1
+  sw.stack: 1
+  sw.stack-variant: 1
+  arch.sw: 1
+output:
+  filename: Dockerfile
+  type: sw.docker-image.stack-device.{{this.children.sw.stack.slug}}
+  slug: "{{this.children.hw.device-type.slug}}-{{this.children.sw.os.slug}}-{{this.children.sw.stack.slug}}"
+  path: "{{this.children.sw.stack.slug}}/{{this.children.hw.device-type.slug}}/{{this.children.sw.os.slug}}/{{this.children.sw.os.version}}/{{this.children.sw.stack.version}}/{{this.children.sw.stack-variant.slug}}"
```

Comment:
Should we include the stack version in the path? Otherwise if there are stacks A v1 and A v2, the second one will override the first, right?

Reply:
It's already there `{{this.children.sw.stack.slug}}/{{this.children.hw.device-type.slug}}/{{this.children.sw.os.slug}}/{{this.children.sw.os.version}}/**{{this.children.sw.stack.version}}**/{{this.children.sw.stack-variant.slug}}`